sw/device/lib/dif/dif_gpio.c

```diff
@@ -70,96 +70,112 @@ void (*abort_if_invalid_fptr)(dif_gpio_t const *const) = abort_if_invalid;
  *
  * @param index Bit index in a 32-bit register.
  */
-static inline uint32_t index_to_mask(uint32_t const index) {
+static inline uint32_t index_to_mask(const uint32_t index) {
   if (index >= (8 * sizeof(uint32_t))) {
     abort();
   }
   return 1U << index;
 }
 #ifdef GTEST_TEST
-uint32_t (*index_to_mask_fptr)(uint32_t const) = index_to_mask;
+uint32_t (*index_to_mask_fptr)(const uint32_t) = index_to_mask;
 #define index_to_mask index_to_mask_fptr
 #endif
 
-static inline uint32_t read_register(dif_gpio_t const *const gpio,
-                                     uint32_t const offset) {
+static inline uint32_t read_register(const dif_gpio_t *const gpio,
```

Comment:
I will remove all `abort()` related code since calling `abort()` was the least favored option during the meeting. If we decide to do null-checks using enums return values, I will make those changes.